meta-mel-support/recipes-bsp/96boards-tools/96boards-tools/resize-helper.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/sh
+### BEGIN INIT INFO
+# Provides: Resize root filesystem to fit available disk space
+# Description: Start resize-helper, then remove own init from runlevels,
+# as it is required only for first boot. udevadm settle provide partition
+# table details required by resize-helper.
+### END INIT INFO
+                                                    
+PATH=/sbin:/bin:/usr/sbin:/usr/bin
+DESC=resize-helper
+                                                                        
+case $1 in
+	start)
+		echo "Starting $DESC"
+		udevadm settle
+		resize-helper
+		
+		echo "unlinking $DESC from runlevels starting"
+		unlink /etc/rc3.d/S22resize-helper.sh &> /dev/null
+		unlink /etc/rc5.d/S22resize-helper.sh &> /dev/null
```

Comment:
The use of the simplified 'unlink' rather than just using rm like everyone else is a bit odd, but admittedly it's SuSv2, so it shouldn't hurt anything.